adhocracy-plus/templates/a4dashboard/includes/project_result_form.html

```diff
@@ -0,0 +1,20 @@
+<div class="row">
+    <div class="col-md-10 col-lg-8">
+      <h5>{{ insight|title|slice:':9' }}</h5>
+    </div>
+
+    <div class="col-md-10 col-lg-12">
+      <p> {{ insight_label }} </p>
+    </div>
+      <p>
+
+    {% for label, value in counts %}
+        <div class="col-md-10 col-lg-8">
+            {{ label|title }}: {{ value }}
+        </div>
+    {% endfor %}
+      </p>
+</div>
+{% include 'a4dashboard/includes/form_field.html' with field=insight_form.display %}
+{% include 'a4dashboard/includes/form_field.html' with field=form.result %}
+
```

Comment:
@philli-m @Rineee any tips how to style the checkbox and make the `Display` form label invisible as in the design?

![image](https://github.com/liqd/adhocracy-plus/assets/5695572/6e921b76-47ca-45e9-a740-4275995c35ea)

Design here:
https://app.zeplin.io/project/5b051f895f8bcd4920567e89/screen/649ae3acb6b6da22f3a82533

Reply:
@m4ra I think you can use the check_box_form_field and that should sort most of the styling issues 
we use it in the login form like this 
            {% include 'a4_candy_contrib/includes/form_checkbox_field.html' with field=form.remember %}

Reply:
uhm for some reason, this template for the check form field doesn't render the form. I will look ihto it again TMR

Reply:
I couldn't make it work with the checkbox_form from the contrib templates, and not enough time to look into it more. Either I look at it on Monday again, or apply some css?